docs/pages/getting-started/providers/threads.mdx

```diff
@@ -0,0 +1,83 @@
+import { Callout } from "nextra/components"
+import { Code } from "@/components/Code"
+
+<img align="right" src="/img/providers/threads.svg" height="64" width="64" />
+
+# Instagram Provider
```

Comment:
Is this headline correct or should it have been "Threads Provider"?

Reply:
Thank you for pointing this out. I agree with your suggestion.
I have created a new PR.
https://github.com/nextauthjs/next-auth/pull/11245

Reply:
Thanks for responding so quickly!